.github/workflows/connect-test-params.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - run: yarn install
       # todo: ideally reuse libs from build step
       - run: yarn build:libs
-      - run: './docker/docker-connect-test.sh node -p "${{ inputs.test-pattern }}" -f 2-latest -i ${{ inputs.methods }}'
```

Comment:
We need to temporarily switch to the latest firmware for testing.

Reply:
could this be possibly replaced by adding a legacy result here? https://github.com/trezor/trezor-suite/blob/develop/packages/connect/e2e/__fixtures__/stellarSignTransaction.ts#L143